reefer_monitoring/v1/reefer_v1.0.0-Beta-1.yaml

```diff
@@ -0,0 +1,524 @@
+openapi: 3.0.0
+x-stoplight:
+  id: a6iz4bb3dw0lc
+info:
+  title: reefer_v1.0.0-Beta-1
+  version: 1.0.0-Beta-1
+  contact:
+    name: Digital Container Shipping Association (DCSA)
+    email: info@dcsa.org
+    url: 'https://dcsa.org'
+  license:
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+    name: Apache 2.0
+  description: |-
+    Exchange data uniformly on data collected from live reefers between operational supply chain partners such as carriers, terminals and vessel operators.
+
+    For explanation to specific values or objects please refer to the [Information Model 2023.Q1](https://dcsa.org/wp-content/uploads/2022/02/DCSA-Information-Model-2023.Q1.pdf).
+
+    There are two ways to get events. Either they can be **polled by you** or they can be **pushed to an endPoint** by the implementor.
+
+    ## Poll model
+    Polling data can be done using
+
+        GET /v1/events
+
+    Calling this endPoint will return a list of events that fulfill the query parameters specified. The list returned is sorted in ascending (`ASC`) order based on the creation date-time of the event (`eventCreatedDateTime`). **This endPoint is mandatory to implement.**
+
+    ## Push model
+    Having events pushed to you is possible via setting up a subscription, **DR to update here the latest push endpoint** 
```

Comment:
Do you need to add a Reefer Operational Notification API for this? You have removed the `push` part?

Reply:
longer term yes as its in the spec from the SME's I have removed for now as we will look to implement the notification API like you shared to me recently.